### PR TITLE
draft for post and logo

### PR DESCRIPTION
**Pull request by shiftkey**

### CODE REVIEW

File: _includes/past.html [hunk 1]
```diff
@@ -12,11 +12,19 @@
...
-	<!--		<div class="Tile one lime"></div>
-			<div class="Tile one lime"></div>
-			<div class="Tile one lime"></div>-->
+			<a href="/pretzel/">
+				<div class="Tile two-h lime">
+					<img src="/img/tiles/pretzel.png" /><h2>pretzel</h2>
+				</div>
+			</a>
+			<a href="/carnac/">
+				<div class="Tile two-h lime">
```
Comment (ghost) [inline _includes/past.html]: should be `one` not `two-h`. metro.css missing.

File: _posts/2012-02-20-carnac-review.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+title: "carnac review"
+author: "@aeoth, @shiftkey and @tobin"
+comments: true
+---
+
+What started out as an excuse to watch [Youtube](http://www.youtube.com/watch?v=9m_dT0wsrGI) clips and make jokes about turbans quickly turned into a neat little tool called Carnac, for those people who want to display their keyboard activities on screen.
+
+<br />
+
+<center>![](/img/carnac-logo.png)</center>
```
Comment (ghost) [inline _posts/2012-02-20-carnac-review.md]: Jekyll won't render it. Center tag around it confuses the inline image. Either drop the center or use an `img` tag

File: _includes/past.html (hunk 1 above)
Comment (shiftkey) [inline _includes/past.html]: Changed to `one` - I won't get to adding metro.css tonight though.

File: _posts/2012-02-20-carnac-review.md (hunk 2 above)
Comment (shiftkey) [inline _posts/2012-02-20-carnac-review.md]: Removed

File: _posts/2012-02-20-github-for-outlook.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ * Display notifications as configured on GitHub.
+ * View and respond to comments from inside Outlook.
+ * Leverage Outlook features like email, labels and perhaps tasks to integrate GitHub work with the rest of your day-to-day work.
+
+
+## The technology under the hood
+
+
+
+* Using @jakeginnivan's VSTO helper to build off as a framework
```
Comment (ghost) [inline _posts/2012-02-20-github-for-outlook.md]: link?

File: _posts/2012-02-20-github-for-outlook.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* WPF App
+* Settings page to hook in to your github api key.
```
Comment (ghost) [inline _posts/2012-02-20-github-for-outlook.md]: Formatting: GitHub API Key

File: _posts/2012-02-20-github-for-outlook.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+ * View and respond to comments from inside Outlook.
+ * Leverage Outlook features like email, labels and perhaps tasks to integrate GitHub work with the rest of your day-to-day work.
+
+
+## The technology under the hood
+
+
+
+* Using @jakeginnivan's VSTO helper to build off as a framework
+* WPF App
```
Comment (ghost) [inline _posts/2012-02-20-github-for-outlook.md]: Are we linking to Milestone for this?

File: _posts/2012-02-20-github-for-outlook.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+* Settings page to hook in to your github api key.
+* Enumerate your projects
+* Create a new issue from an email, assign it against a project, apply a label
+
+
+* Could also create a task in outlook for that, link back to the issue in Github
```
Comment (ghost) [inline _posts/2012-02-20-github-for-outlook.md]: GitHub

**Pull request merged**